src/schemathesis/specs/openapi/examples.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict, List, Optional
+
+from hypothesis.strategies import SearchStrategy
+
+from ..._hypothesis import PARAMETERS, _get_case_strategy, prepare_strategy
+from ...models import Case, Endpoint
+
+
+def get_param_examples(endpoint_def: Dict[str, Any], examples_field: str) -> List[Dict[str, Any]]:
+    all_parameter_examples = []
+    for param in endpoint_def.get("parameters", []):
+        if examples_field in param:
+            # add value to list of params with examples
+            param_type = {
+                "body": "body",
+                "cookie": "cookies",
+                "header": "headers",
+                "path": "path_parameters",
+                "query": "query",
+            }.get(param["in"])
+            assert param_type in PARAMETERS
+            all_parameter_examples.append(
+                {
+                    "type": param_type,
+                    "name": param["name"],
+                    "examples": [example["value"] for example in param[examples_field].values()],
+                }
+            )
+    return all_parameter_examples
+
+
+def get_request_body_examples(endpoint_def: Dict[str, Any], examples_field: str) -> Dict[str, Any]:
+    request_bodies_items = endpoint_def.get("requestBody", {}).get("content", {}).items()
+    if not request_bodies_items:
+        return {}
+    # first element in tuple in media type, second element is dict
+    media_type, schema = next(iter(request_bodies_items))
+    param_type = "body" if not media_type == "multipart/form-data" else "form_data"
```

Comment:
A more idiomatic way would be:
```suggestion
    param_type = "body" if media_type != "multipart/form-data" else "form_data"
```